cpp/arcticdb/util/error_code.hpp

```diff
@@ -57,7 +59,8 @@ inline std::unordered_map<ErrorCategory, const char*> get_error_category_names()
     ERROR_CODE(3000, E_NO_SUCH_VERSION)  \
     ERROR_CODE(4000, E_DESCRIPTOR_MISMATCH)  \
     ERROR_CODE(5000, E_KEY_NOT_FOUND) \
-    ERROR_CODE(5001, E_DUPLICATE_KEY)
+    ERROR_CODE(5001, E_DUPLICATE_KEY) \
+    ERROR_CODE(6000, E_UNSORTED_DATA)
```

Comment:
Please leave a `\` at the end so we don't end up touching the previous line each time we add one